config_files/training/config_example_coca.yaml

```diff
@@ -170,7 +170,23 @@ wrapped_model:
     sharding_strategy: FULL_SHARD
     block_names: [TransformerBlock, VisionTransformerBlock]
 
-model:
+model: 
+  component_key: model
+  variant_key: weight_initialized
+  config:
+    model:
+      instance_key: model_raw
+      pass_type: BY_REFERENCE
+    weight_initializer:
+      component_key: weight_initialization
+      variant_key: composed
+      config:
+        model_type: coca
+        weight_init_type: plain
+        mean: 0.0
+        plain_std: 0.02
```

Comment:
```suggestion
        std: 0.02
```

Reply:
Fixed in [0e29447](https://github.com/Modalities/modalities/pull/168/commits/0e29447002713b84d5bf6c0057b33b964e96cd82)